### PR TITLE
New syntax

### PR DESCRIPTION
**Pull request by wyd3x**
Update to new syntax (now const saved, my bad last time)

### CODE REVIEW

**Comment by asherkin** [comment]
In the future please update your PRs rather than creating additional ones.

File: plugins/include/clients.inc [hunk 1]
```diff
@@ -301,14 +301,14 @@ native bool:GetClientIP(client, String:ip[], maxlen, bool:remport=true);
  * @param client		Player index.
  * @param auth			Buffer to store the client's auth string.
  * @param maxlen		Maximum length of string buffer (includes NULL terminator).
- * @param validate		Check backend validation status.
- * 				DO NOT PASS FALSE UNLESS YOU UNDERSTAND THE CONSEQUENCES,
+ * @param validate		Check backend validation status.
+ * 				DO NOT PASS FALSE UNLESS YOU UNDERSTAND THE CONSEQUENCES,
```
Comment (50Wliu) [inline plugins/include/clients.inc]: Why were these two lines changed?

File: plugins/include/clients.inc [hunk 2]
```diff
@@ -496,7 +496,7 @@ native AddUserFlags(client, AdminFlag:...);
  * @noreturn
  * @error				Invalid client index, or client not connected.
  */
-native RemoveUserFlags(client, AdminFlag:...);
+native void RemoveUserFlags(int client, AdminFlag:...);
```
Comment (50Wliu) [inline plugins/include/clients.inc]: This native and the one above it are still using old-style `AdminFlag:...` syntax.

**Comment by psychonic** [comment]
Thank you for the contribution!

We instead decided to go with PRs #509, #510, and #511 for this.

**Pull request closed without merging**